packages/graphql-server-v2/src/server.ts

```diff
@@ -1,50 +1,61 @@
 import express, { Express } from 'express'
 import { graphqlHTTP } from 'express-graphql'
 import { createSchema, CallBackendArguments } from 'swagger-to-graphql'
-import { createPlatformAxiosInstance } from './axios'
 import 'isomorphic-fetch'
 import graphqlHeader from 'express-graphql-header'
 import { API_VERSION } from './constants'
+import cors from 'cors'
+import * as Sentry from '@sentry/node'
+import { AxiosInstance } from 'axios'
+import config from './../config.json'
 
-const handlePlatformCall = async ({ context, requestOptions }: CallBackendArguments<Request>) => {
-  if (!(context.headers as any).authorization) {
-    return {
-      statusCode: 401,
-    }
-  }
+if (config.SENTRY_DSN) {
+  Sentry.init({ dsn: config.SENTRY_DSN })
+}
 
-  const axios = createPlatformAxiosInstance()
-  try {
-    const result = await axios[requestOptions.method](requestOptions.path, {
-      headers: {
-        Authorization: (context.headers as any).authorization,
-        API_VERSION,
-      },
-      body: requestOptions.body,
-    })
+const handlePlatformCall =
+  (axios: AxiosInstance) =>
+  async ({ context, requestOptions }: CallBackendArguments<Request>) => {
+    if (!(context.headers as any).authorization) {
+      return {
+        statusCode: 401,
+      }
+    }
+    try {
+      const result = await axios[requestOptions.method](
+        `${requestOptions.path}${
+          requestOptions.query ? '?' + new URLSearchParams(requestOptions.query as any).toString() : ''
+        }`,
+        {
+          headers: {
+            Authorization: (context.headers as any).authorization,
+            API_VERSION,
+          },
+          body: requestOptions.body,
+        },
+      )
 
-    return result.data
-  } catch (e: any) {
-    console.error(e)
-    return {
-      error: e.message,
+      return result.data
+    } catch (e: any) {
+      console.error(e)
+      return {
+        error: e.message,
+      }
     }
   }
-}
-
-export const bootstrap = async (): Promise<Express> => {
-  console.log('Fetching swagger doc...')
-  const swaggerResponse = await fetch(`${process.env.PLATFORM_API_BASE_URL}/docs/swagger/agencyCloud_swagger.json`)
-
-  const swagger = await swaggerResponse.json()
-  console.log('Starting GQL server')
 
+export const bootstrap = async (axiosInstance: AxiosInstance): Promise<Express> => {
   const schema = await createSchema({
-    swaggerSchema: swagger as any,
-    callBackend: handlePlatformCall,
+    swaggerSchema: `${config.PLATFORM_API_BASE_URL}/docs/swagger/agencyCloud_swagger.json`,
```

Comment:
Is the plan still to generate this on a cron job rather than generating the schema on the fly each time?

Reply:
am not sure it's possible in the way that we planned, I was going to attempt a different method today. Currently, using the package I can route all resolvers to a singular function whereas generating the schema I'd have to bypass this package in the codebase and attempt to setup apollo etc